src/documents/Enactors.md

```diff
@@ -23,18 +23,18 @@ Each of these is explained below.
 
 ### Grade ###
 
-Enactors must be defined using the `fluid.prefs.enactors` [grade](ComponentGrades.md), as shown in the following code block:
+Enactors must be defined using the `fluid.prefs.enactor` [grade](ComponentGrades.md), as shown in the following code block:
```

Comment:
@cindyli, could you start a document listing API changes from 1.5 to ... well, I guess it would be 2.0, wouldn't it? Things like this enactor name change, the xxRelayComponent name changes, and anything else you know of. It doesn't have to be super final, just a place where we start recording this so we don't forget, and so we don't have a huge task trying to remember everything when release time comes